spec/services/adapters/structs/capital_details_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
```

Comment:
I didn't change anything in this struct but suddenly rubocop noticed there was no code coverage